Dangerfile

```diff
@@ -77,3 +77,11 @@ end
 
 # Run SwiftLint
 swiftlint.lint_files
+
+# Xcode summary
+xcode_summary.ignored_results { |result|
+  result.message.start_with?('ld') # Ignore ld_warnings
```

Comment:
Why?

Reply:
I'm not sure if it's important to show ld linker warnings, and since there are some of them in the builds I just skip them :))
Maybe I can try to fix them in this and remove this 😊

Reply:
@BasThomas The ld: warnings of the project seems to have something to do with the libraries that are imported via Carthage. Maybe I'm missing something but as far as I could understand the libraries have to set the target’s “Require Only App-Extension-Safe API” build setting to Yes. If you don’t, Xcode reminds you to do so by displaying the warning “linking against dylib not safe for use in application extensions”. You can see more [here](https://developer.apple.com/library/archive/documentation/General/Conceptual/ExtensibilityPG/ExtensionScenarios.html)
Because of that I just Ignore them :))

Reply:
Hm, I am not so sure about this one. Moya and its dependencies should definitely be extension-safe APIs. What does the warning output look like if we remove this?

Reply:
I can remove this so you can see the logs :)) I'll ping you then